analyses/cell-type-consensus/exploratory-notebooks/04-brain-and-cns-consensus-celltypes.Rmd

```diff
@@ -121,6 +116,8 @@ all_results_df <- results_files |>
 # get all possible cell types 
 # there are exactly 26 of them so we will use alphabet with two greys for unknown/all remaining 
 unique_celltypes <- all_results_df |> 
+  # order by frequency otherwise neurons will be light grey 
+  dplyr::arrange(desc(total_cells_per_annotation)) |> 
```

Comment:
Do you want to add this to all of the plots? It seems reasonable, but maybe not necessary for these explorations.

Reply:
What do you mean? This just adjusts the colors for the plots in this notebook so that neuron doesn't get assigned to the light grey color in the `alphabet` palette. Its present in all the plots in this notebook, but isn't applicable to plots in the other notebooks.

Reply:
I wondered if it makes sense to always arrange the colors from most to least frequent?

Reply:
We already do that in the exploration notebook that looks at all samples. The only one we don't is osteo and I just tried it and it looked worse, so I'm going to keep the colors there.